ext/mysqli/php_mysqli_structs.h

```diff
@@ -121,6 +121,13 @@ struct st_mysqli_warning {
    	MYSQLI_WARNING	*next;
 };
 
+typedef struct st_mysqli_sql_exception MYSQLI_SQL_EXCEPTION;
+struct st_mysqli_sql_exception {
+	zend_string	*message;
+	zend_string	*sqlstate;
+	int			code;
+};
```

Comment:
What's up with this change?

Reply:
Oops, this should not be here. This was just me experimenting with code.